omp/__init__.py

```diff
@@ -41,18 +43,43 @@ def init_msvc(self, ver):
             self.libomp = ctypes.CDLL(vcomp_path)
             self.version = 20
 
+    def get_libomp_names(self):
+        if cxx is None:
+            # Can't tell what compiler we're using, guessing we need libgomp
+            return 'libgomp'
```

Comment:
shouldn't that be `['libgomp']` to always return a list ?

Reply:
Yes indeed. Untested paths are dangerous .... fixed now.